person_class.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name = 'UNKNOWN', parent_permission: true)
+    @id = id
```

Comment:
- [ ] **[OPTIONAL]** We can generate the `@id` so we don't need to pass it.

  ```
  def initialize(age, name = 'UNKNOWN', parent_permission: true)
    @id = rand(1...1000)
  ```

Reply:
yeah, some special id gen. can be used.